package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "start": "node ./server/bin.js",
-    "test": "grunt test"
+    "test": "grunt test",
+    "postinstall": "./node_modules/bower/bin/bower install"
```

Comment:
Maybe you can put it into `app.json`: https://devcenter.heroku.com/articles/app-json-schema#scripts